session.go

```diff
@@ -1,97 +1,99 @@
 package bond
 
 import (
+	"database/sql"
+	"fmt"
 	"reflect"
 	"sync"
 
-	"upper.io/db"
+	"upper.io/db.v2"
+	"upper.io/db.v2/lib/sqlbuilder"
```

Comment:
this PR looks amazing and feels so close.. it's probably good enough for merge, but any chance we can think of a way to not include sqlbuilder here directly..? what is the reason exactly?